src/sidebar/store/modules/selection.js

```diff
@@ -51,20 +45,26 @@ TAB_SORTKEYS_AVAILABLE[uiConstants.TAB_ORPHANS] = [
   'Location',
 ];
 
+/**
+ * Utility function that returns all of the properties of an object whose
+ * value is `true`.
+ *
+ * @param {Object} obj
+ * @return {any[]}
+ */
+function truthyKeys(obj) {
```

Comment:
I realized this name is a little silly: it doesn't return for "truthy" entries, it requires literal `true`. I copied it from elsewhere...